sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -496,6 +497,12 @@ export class AssessmentController {
         ? output[outputName] + contributionTotal.total
         : contributionTotal.total;
     });
+    programYearTotal.costTotals?.forEach((costTotal) => {
```

Comment:
As we discussed, let's rename and expand the current `contributionTotals` to include these new properties and allow its expansion in the future.

Reply:
I noticed it was renamed to `costAndContributionTotals`. Is there a problem trying to keep it more generic and avoid renaming it in the future?
Was there a problem with the suggested `workflowOuputs`?

Reply:
I didn't see you added a suggestion. Changed the var name.